OctoKit/Issue.swift

```diff
@@ -58,25 +104,109 @@ public extension Octokit {
             }
         }
     }
+    
+    /**
+     Fetches an issue in a repository
+     - parameter owner: The user or organization that owns the repository.
+     - parameter repository: The name of the repository.
+     - parameter number: The number of the issue.
+     - parameter completion: Callback for the outcome of the fetch.
+     */
+    public func issue(owner: String, repository: String, number: Int, completion: (response: Response<Issue>) -> Void) {
```

Comment:
I must be misunderstanding what these method parameters are used for. I was thinking it would be nice for the compiler to catch mistakes where client code might pass in `"Crash when fooing bar"` or `"#2"` for the issue number; better to catch such problems at build time then wait until the API returns a 404 at runtime.

Are there reasons to prefer a string in this case, other than aesthetics? This PR works as-is without nerdishbynature/RequestKit#11; that PR is only about encoding the labels and milestone in the JSON-encoded PUT request.